sklearn/inspection/_partial_dependence.py

```diff
@@ -467,23 +450,27 @@ def plot_partial_dependence(estimator, X, features, feature_names=None,
         `method` is 'brute'.
 
     features : list of {int, str, pair of int, pair of str}
-        The target features for which to create the PDPs.
-        If features[i] is an int or a string, a one-way PDP is created; if
-        features[i] is a tuple, a two-way PDP is created. Each tuple must be
-        of size 2.
+        The target features for which to create the PDP/ICEs.
         if any entry is a string, then it must be in ``feature_names``.
 
+    calc_method : partial_dependence or individual_conditional_expectation
+        The function to calculate values for the PDP/ICE plot
+
+    display_class : PartialDependenceDisplay or
```

Comment:
Basically we don't need both `calc_method` and `display_class` since we don't want to make a mix `ice/pd`
Maybe we can introduce the proposed keyword by @jnothman `individual` which will be a bool.
Internally we will choose the method to call with the display associated.

Reply:
Only saw this now. I've used `plot_type` which is a bit more explicit I suppose. Happy to use `individual` if you find it better.

https://github.com/scikit-learn/scikit-learn/pull/16164/commits/cb54fbd95fefb5a89a6b395c26f11b76ec4f1f9d#diff-0c8623ee957e0256d29ac53830094281R453-R454